src/elf/compression_header.rs

```diff
@@ -30,7 +30,7 @@ pub const ELFCOMPRESS_HIPROC: u32 = 0x7fff_ffff;
 macro_rules! elf_compression_header_std_impl { ($size:ty) => {
 
     #[cfg(test)]
-    mod test {
+    mod tests {
```

Comment:
What is the problem with the old name?

Reply:
It's just a convention from [the Book](https://doc.rust-lang.org/book/ch11-03-test-organization.html#the-tests-module-and-cfgtest).